### ISSUE TITLE
Vanilla branch?

### ISSUE THREAD

**Issue by koutsie** [SwagSoftware/Kisak-Strike#11]
Shouldn't there be a vanilla branch that doesnt have kisakui etc :D

**Comment by LWSS** [comment]
yes there will be a vanilla branch soon that will be unchanged from regular CSGO. I plan to make some game-breaking changes like making it so you can buy the silenced m4a1 and the regular at the same time.
